lib/moku/release.rb

```diff
@@ -26,18 +26,24 @@ def initialize(artifact:, deploy_config:, remote_runner: nil, release_dir: nil)
     def_delegators :@artifact, :path
     def_delegators :@deploy_config, :systemd_services, :sites
 
+    # Absolute path to the directory where all releases are stored
     def releases_path
       deploy_config.deploy_dir/"releases"
     end
 
+    # Absolute path where this release will be deployed
     def deploy_path
       releases_path/release_dir
     end
 
+    # Absolute, logical path to where the application lives
     def app_path
       deploy_config.deploy_dir/"current"
     end
 
+    # Run a command on this release's hosts, after applying the given scope
+    # @param scope [Sites::Scope]
+    # @param command [String]
     def run(scope, command)
```

Comment:
It looks like these comments aren't related to the PR but are more just housekeeping.

Reply:
Yes, see 8d72c242637e3e237bf58a20730d55140de0b590